docs/cicd/cicd-spans.md

```diff
@@ -0,0 +1,124 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Spans
+--->
+
+# Semantic conventions for CICD spans
+
+**Status**: [Development][DocumentStatus]
+
+<!-- toc -->
+
+- [CICD Spans](#cicd-spans)
+  - [Pipeline run](#pipeline-run)
+  - [Pipeline task run](#pipeline-task-run)
+
+<!-- tocstop -->
+
+## CICD Spans
+
+The conventions described in this section are specific to Continuous Integration / Continuous Deployment (CICD) systems.
+
+Any resources of the [CICD and VCS resource conventions][cicdres] that apply SHOULD be used.
+
+[cicdres]: /docs/resource/cicd.md (CICD and VCS resource conventions)
+
+### Pipeline run
+
+For all pipeline runs, a span with kind `SERVER` SHOULD be created corresponding to the execution of the pipeline run.
+
+The span name MUST follow the overall [guidelines for span names](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.43.0/specification/trace/api.md#span).
+
+The span name SHOULD be `{action} {pipeline}` if there is a (low-cardinality) `pipeline` available.
+If there is no (low-cardinality) `{pipeline}` available, the span name SHOULD be `{action}`.
+
+The `{action}` SHOULD be one of the following values: `BUILD`, `RUN`, `SYNC`. If none of these values applies a custom value MAY be used.
```

Comment:
any reason for ALLCAPS? Also, would we have action appearing on metrics in some shape?  We probably want a dedicated attribute for it.

Reply:
I used ALLCAPS because I based this on the the http semconv which uses the http method (ie `GET`, `POST`, ...) and because Jenkins already uses `BUILD` in its pipeline run span name.

I think it makes sense to use ALLCAPS because it clearly distinguishes the action from the pipeline name.

I will make it an enum attribute, since it also makes it easier to have consistent language for the listed values and add additional ones in the future.
I don't think we will have this attribute on metrics.

Reply:
Made `{action}` an attribute: 3cefced99c742652925188a3acbf1bfea858297f